torat_client/netclient.go

```diff
@@ -51,14 +51,14 @@ func NetClient() {
 
 	t, err := tor.Start(nil, &conf)
 	if err != nil {
-		log.Println("[!] Tor could not be started:", err)
+		log.Println("[NetClient] [!] Tor could not be started: ", err)
```

Comment:
All of the Println functions already add a space between arguments, so please remove the trailing space.

(there are also other cases)

Reply:
This is def my bad, I keep adding these out of force of habit